qemu/tests/cfg/virtio_fs_set_capability.cfg

```diff
@@ -0,0 +1,81 @@
+- virtio_fs_set_capability:
+    no RHEL.6 RHEL.7 RHEL.8.0 RHEL.8.1
+    no Win2008 Win7
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8.u0 Host_RHEL.m8.u1
+    type = virtio_fs_set_capability
+    virt_test_type = qemu
+    required_qemu = [4.2.0,)
+    s390, s390x:
+        required_qemu = [5.2.0,)
+    kill_vm = yes
+    start_vm = no
+    fs_driver = virtio-fs
+    fs_source_dir = /var/lib/avocado/data/avocado-vt/virtio_fs_test/
+    force_create_fs_source = no
+    remove_fs_source = yes
+    fs_target = 'myfs'
+    fs_driver_props = {"queue-size": 1024}
+    mem = 4096
+    mem_devs = mem1
+    backend_mem_mem1 = memory-backend-file
+    mem-path_mem1 = /dev/shm
+    size_mem1 = 4G
+    use_mem_mem1 = no
+    share_mem = yes
+    !s390, s390x:
+        guest_numa_nodes = shm0
+        numa_memdev_shm0 = mem-mem1
+        numa_nodeid_shm0 = 0
+    io_timeout = 600
+    fs_dest = '/mnt/${fs_target}'
+    driver_name = viofs
+    Windows:
+        # install winfsp tool
+        i386, i686:
+            install_path = 'C:\Program Files'
+        x86_64:
+            install_path = 'C:\Program Files (x86)'
+        install_cmd = 'msiexec /i WIN_UTILS:\winfsp-1.7.20172.msi /qn'
+        check_installed_cmd = 'dir "%s" |findstr /I winfsp'
+        start_vfs_cmd = "start /b %s -d -1 -D C:\\viofs_log.txt"
+        check_virtiofs_cmd = 'wmic process where caption="virtiofs.exe" list brief'
+        virtio_win_media_type = iso
+        cdroms += " virtio"
+    variants:
+        - with_cache:
+            variants:
+                - auto:
+                    fs_binary_extra_options = " -o cache=auto"
+                - always:
+                    fs_binary_extra_options = " -o cache=always"
+                - none:
+                    fs_binary_extra_options = " -o cache=none"
+    variants:
+        - remove_capability:
+            cmd_create_fs_source = 'mkdir -p /tmp/virtio_fs1_test'
+            cmd_run_virtiofsd = '/usr/libexec/virtiofsd --socket-path=%s'
+            cmd_dd = 'dd if=/dev/urandom of=%s bs=1M count=2048 oflag=direct'
+            cmd_md5 = 'md5sum %s'
+            io_timeout = 120
+            Windows:
+                cmd_dd = 'dd if=/dev/random of=%s bs=1M count=200'
+                cmd_md5 = "%s: && md5sum.exe %s"
+            cmd_capsh_print = 'capsh --print'
+            cmd_capsh_drop = 'capsh --drop=%s --'
+            variants:
+                - cap_dac_read_search:
+                    only virtio_fs_set_capability..with_cache.auto
```

Comment:
the two scenarios have the specified cache, how about adding fs_binary_extra_options in each variant.
If later we have another property or need to test all cache,then add them

Reply:
As you can see from line 46, the requirements for future expansion are taken into account here. If you want to add `fs_binary_extra_options` in the future, just remove only.